pkg/scenarioserver/scenarioserver.go

```diff
@@ -151,14 +120,54 @@ func (s ScenarioServer) getPreparedScenarioStepById(id string, step int) (Prepar
 	return PreparedScenarioStep{}, fmt.Errorf("error while retrieving scenario step, most likely doesn't exist in index")
 }
 
-func (s ScenarioServer) getPreparedScenarioById(id string) (PreparedScenario, error) {
+func (s ScenarioServer) getPrintableScenarioIds(accessCodes []string) []string {
+	var printableScenarioIds []string
+	var printableCourseIds []string
+	for _, acString := range accessCodes {
+		ac, err := s.acClient.GetAccessCode(acString, false)
+		if err != nil {
+			glog.Errorf("error retrieving access code: %s %v", acString, err)
+		} else {
```

Comment:
instead of introducing an `else` here, can we place a `continue` statement in the err handling block above? it will improve readability

Reply:
Thanks for the hint @ebauman! You are right, it now looks much better. I could get rid of all the else blocks within loops, tested it and it still works fine :)